src/modules/list.js

```diff
@@ -0,0 +1,44 @@
+// const result = [
+//   {
+//     user: 'Name: ',
+//     score: 100,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 20,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 50,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 78,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 125,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 77,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 42,
+//   },
+// ];
+
+// const scoreList = () => {
```

Comment:
- [ Optional] Kindly remove the commented code, your app will look more professional.